src/coreclr/jit/codegencommon.cpp

```diff
@@ -2075,7 +2075,8 @@ void CodeGen::genEmitMachineCode()
         {
             printf("; ============================================================\n\n");
         }
-        printf(""); // in our logic this causes a flush
+
+        fflush(jitstdout());
```

Comment:
So the comment above was wrong? (about flushing)

Reply:
Perhaps? I could not find where `jitstdout` got flushed (it is set to unbuffered in some cases).